community/portworx/templates/portworx-storageclasses.yaml

```
@@ -54,7 +54,7 @@ parameters:
 # Please refer to : https://docs.portworx.com/scheduler/kubernetes/dynamic-provisioning.html
 #
 kind: StorageClass
-apiVersion: storage/v1
+apiVersion: storage.k8s.io/v1beta1
```

This should be the GA version of this spec:
```suggestion
apiVersion: storage.k8s.io/v1
```